<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>399</width>
    <height>524</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QGridLayout" name="gridLayout">
    <item row="0" column="0">
     <widget class="QStackedWidget" name="stackedWidget">
      <property name="currentIndex">
       <number>0</number>
      </property>
      <widget class="QWidget" name="listPage">
       <layout class="QGridLayout" name="gridLayout_2">
        <item row="0" column="0">
         <widget class="QListWidget" name="listWidget"/>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="detailPage">
       <layout class="QGridLayout" name="gridLayout_3">
        <item row="14" column="1" colspan="4">
         <widget class="QDateEdit" name="dateEdit">
          <property name="dateTime">
           <datetime>
            <hour>0</hour>
            <minute>0</minute>
            <second>0</second>
            <year>2019</year>
            <month>1</month>
            <day>1</day>
           </datetime>
          </property>
          <property name="calendarPopup">
           <bool>true</bool>
          </property>
         </widget>
        </item>
        <item row="3" column="2">
         <widget class="QCheckBox" name="ceEdit">
          <property name="text">
           <string>CE</string>
          </property>
         </widget>
        </item>
        <item row="3" column="4">
         <widget class="QCheckBox" name="eoEdit">
          <property name="text">
           <string>EO</string>
          </property>
         </widget>
        </item>
        <item row="0" column="0">
         <widget class="QLabel" name="label">
          <property name="text">
           <string>Name:</string>
          </property>
         </widget>
        </item>
        <item row="1" column="0">
         <widget class="QLabel" name="label_2">
          <property name="text">
           <string>E-mail:</string>
          </property>
         </widget>
        </item>
        <item row="1" column="1" colspan="4">
         <widget class="QLineEdit" name="emailEdit"/>
        </item>
        <item row="15" column="1" colspan="4">
         <widget class="QLineEdit" name="phoneEdit"/>
        </item>
        <item row="2" column="1" colspan="4">
         <widget class="QComboBox" name="examEdit">
          <item>
           <property name="text">
            <string>TEF</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>TEFaQ</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>TEFCA</string>
           </property>
          </item>
         </widget>
        </item>
        <item row="14" column="0">
         <widget class="QLabel" name="label_4">
          <property name="text">
           <string>Exam Date :</string>
          </property>
         </widget>
        </item>
        <item row="16" column="0">
         <widget class="QLabel" name="label_6">
          <property name="text">
           <string>Status :</string>
          </property>
         </widget>
        </item>
        <item row="16" column="1" colspan="4">
         <widget class="QComboBox" name="statusEdit">
          <property name="editable">
           <bool>false</bool>
          </property>
          <property name="currentIndex">
           <number>0</number>
          </property>
          <item>
           <property name="text">
            <string>Waiting Documents</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>Waiting Payment</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>Waiting Confirmation</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>Confirmed</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>Completed</string>
           </property>
          </item>
         </widget>
        </item>
        <item row="15" column="0">
         <widget class="QLabel" name="label_5">
          <property name="text">
           <string>Phone :</string>
          </property>
         </widget>
        </item>
        <item row="2" column="0">
         <widget class="QLabel" name="label_3">
          <property name="text">
           <string>Exam:</string>
          </property>
         </widget>
        </item>
        <item row="17" column="0" colspan="9">
         <widget class="QDialogButtonBox" name="buttonBox">
          <property name="standardButtons">
           <set>QDialogButtonBox::Discard|QDialogButtonBox::Reset|QDialogButtonBox::Save</set>
          </property>
         </widget>
        </item>
        <item row="3" column="1">
         <widget class="QCheckBox" name="eeEdit">
          <property name="text">
           <string>EE</string>
          </property>
         </widget>
        </item>
        <item row="0" column="1" colspan="4">
         <widget class="QLineEdit" name="nameEdit"/>
        </item>
        <item row="3" column="3">
         <widget class="QCheckBox" name="coEdit">
          <property name="text">
           <string>CO</string>
          </property>
         </widget>
        </item>
        <item row="3" column="5">
         <widget class="QCheckBox" name="qcmEdit">
          <property name="text">
           <string>QCM</string>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>399</width>
     <height>21</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuEntries">
    <property name="title">
     <string>Entries</string>
    </property>
    <addaction name="actionAdd"/>
    <addaction name="actionEdit"/>
    <addaction name="actionRemove"/>
   </widget>
   <addaction name="menuEntries"/>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <action name="actionAdd">
   <property name="text">
    <string>Add</string>
   </property>
  </action>
  <action name="actionEdit">
   <property name="text">
    <string>Edit</string>
   </property>
  </action>
  <action name="actionRemove">
   <property name="text">
    <string>Remove</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
